src/components/Calculator.js

```diff
@@ -1,41 +1,40 @@
 /* eslint-disable react/prefer-stateless-function */
 
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import './Calculator.css';
 import calculate from '../logic/calculate';
 
-export default class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      next: null,
-      operation: null,
-      total: null,
-    };
-    this.clickHandler = this.clickHandler.bind(this);
-  }
+const Calculator = () => {
+  // constructor(props) {
+  //   super(props);
+  //   this.state = {
+  //     next: null,
+  //     operation: null,
+  //     total: null,
+  //   };
+  //   this.clickHandler = this.clickHandler.bind(this);
+  // }
```

Comment:
- [ ] Please kindly remove all comments from your codebase as it pollutes the codebase

Reply:
Whoops 😅

Reply:
- [ ] Please as mentioned by the previous reviewer kindly remove the comments line of code to have a clean codebase, if you think you'll need these codes you will need just to check your commits 🙃 🙃